saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleState.java

```diff
@@ -460,9 +474,19 @@ public void notifyMessage(SaicMessage message) throws MqttException {
     notifyCarActivityTime(message.getMessageTime(), false);
   }
 
-  public boolean isRecentlyActive() {
-    return hvBatteryActive
-        || lastCarActivity.isAfter(ZonedDateTime.now().minus(15, ChronoUnit.MINUTES));
+  public boolean shouldRefresh() {
+    switch(refreshMode) {
+      case OFF:
+        return false;
+      case FORCE:
+        setRefreshMode(PERIODIC);
```

Comment:
FORCE should leave the previous refresh mode as is, so that you can set it to OFF for no automatic refreshes, but still send a FORCE from time to time for a manual update

Reply:
OK, that is a good point, will check it out, I assumed periodic, but OFF is also a possibility...